src/components/views/auth/AuthPage.tsx

```diff
@@ -18,20 +18,20 @@ limitations under the License.
 
 import React from 'react';
 import * as sdk from '../../../index';
-import {replaceableComponent} from "../../../utils/replaceableComponent";
 
-@replaceableComponent("views.auth.AuthPage")
```

Comment:
without something to the effect of this decorator this will not work as AuthPage will no longer be replaceable. i.e breaking VectorAuthPage in element-web

Reply:
I'm not *entirely* sure how I would do that in a functional component.

Reply:
I suggest not replacing this file for now